test/middleware/spec.back-link.js

```diff
@@ -1,11 +1,18 @@
-var backLinks = require('../../lib/middleware/back-links');
-
-var util = require('util'),
+var proxyquire = require('proxyquire'),
+    util = require('util'),
     EventEmitter = require('events').EventEmitter;
 
+var getBackLinks = function (previousSteps) {
+    return proxyquire('../../lib/middleware/back-links', {
+        '../util/helpers': {
+            getPreviousSteps: sinon.stub().returns(previousSteps)
+        }
+    });
+};
+
```

Comment:
@joefitter I think that sounds correct. I.e. by `require`ing the helpers and stubbing the method on the object returned, you are only stubbing the method in the scope of that `require` call, but not affecting the return value of the method when it is `require`'d inside `middleware/back-links'`